src/apis/index.js

```diff
@@ -138,8 +140,15 @@ export function postSkillFeedback(payload) {
   });
 }
 
+export function getUserIP() {
+  return ajax.get(`${USER_IP_URL}`, {
+    token: 'b3398b832bf0d7',
```

Comment:
Please use double quotes.

Reply:
@samagragupta The linter apparently changes it back to single quotes.